.travis.yml

```diff
@@ -5,20 +5,13 @@ cache: pip
 
 python:
   - "3.5"
-  - "2.7"
 
 branches:
   only:
     - master
 
 env:
-  - DJANGO="django>=2.1,<2.3"
-  - DJANGO="django>=1.11,<1.12"
```

Comment:
It looks like you are planning to drop django 1.11,which supports up until python 3.7. 
So can you please share why you are dropping it, is it not compatible?